README.md

```diff
@@ -72,7 +72,7 @@ const treeNodes = [
 const getNodeData = (node, nestingLevel) => ({
   data: {
     id: node.id.toString(), // mandatory
-    isLeaf: node.children.length === 0,
+    isLeaf: !!node.children,
```

Comment:
I guess, it should be reversed: a leaf element is an element without children. For the documentation purposes, I suggest using a more explicit check:
```suggestion
    isLeaf: node.children == null || node.children.length === 0,
```